inference-engine/tests_deprecated/functional/vpu/myriad_tests/myriad_configs_tests.cpp

```diff
@@ -81,49 +81,17 @@ TEST_P(myriadIncorrectModelsConfigsTests_nightly, LoadNetworkWithIncorrectConfig
 //  Tests initiation
 //------------------------------------------------------------------------------
 
-static const std::vector<config_t> myriadCorrectPlatformConfigValues = {
-        {{VPU_MYRIAD_CONFIG_KEY(PLATFORM), VPU_MYRIAD_CONFIG_VALUE(2450)}},
-        {{VPU_MYRIAD_CONFIG_KEY(PLATFORM), VPU_MYRIAD_CONFIG_VALUE(2480)}},
-        {{VPU_MYRIAD_CONFIG_KEY(PLATFORM), ""}},
-        // Deprecated
-        {{VPU_CONFIG_KEY(PLATFORM), VPU_CONFIG_VALUE(2450)}},
-        {{VPU_CONFIG_KEY(PLATFORM), VPU_CONFIG_VALUE(2480)}},
-        {{VPU_CONFIG_KEY(PLATFORM), ""}}
-};
```

Comment:
Did you move these tests to new infra>

Reply:
No, I just remove tests related to PLATFORM config since it is deprecated

Reply:
I don't think it's a good idea. Deprecated does not mean not supported, so should work. To ensure it should be tested.

Reply:
I generally agree with Gleb. Also I don't see from the changes that both new names and deprecated names are supported. How can we verify that both sets of names are working after this change?

Reply:
Deprecated test cases were leaved